### PR TITLE
Speech samples

### PR DESCRIPTION
**Pull request by jabubake**
Note : not to be merged till 
https://github.com/GoogleCloudPlatform/google-cloud-java/tree/ml-update-branch is merged.

### CODE REVIEW

**Comment by jabubake** [comment]
@dizcology @lesv Please hold off on review till I resolve the conflicts with master.

File: speech/cloud-client/src/main/java/com/example/speech/QuickstartSample.java [hunk 1]
```diff
@@ -18,13 +18,13 @@
...
 // [START speech_quickstart]
 // Imports the Google Cloud client library
-import com.google.cloud.speech.v1.RecognitionAudio;
-import com.google.cloud.speech.v1.RecognitionConfig;
-import com.google.cloud.speech.v1.RecognitionConfig.AudioEncoding;
-import com.google.cloud.speech.v1.RecognizeResponse;
-import com.google.cloud.speech.v1.SpeechClient;
-import com.google.cloud.speech.v1.SpeechRecognitionAlternative;
-import com.google.cloud.speech.v1.SpeechRecognitionResult;
+import com.google.cloud.speech.v1_1beta1.SpeechClient;
```
Comment (jabubake) [inline speech/cloud-client/src/main/java/com/example/speech/QuickstartSample.java]: Note : this may change

Comment (lesv) [reply speech/cloud-client/src/main/java/com/example/speech/QuickstartSample.java]: hope so.

File: speech/cloud-client/src/main/java/com/example/speech/QuickstartSample.java [hunk 2]
```diff
@@ -33,39 +33,43 @@
...
 
 public class QuickstartSample {
+
+  /**
+   * Demonstrates using the Speech API to transcribe an audio file.
+   */
   public static void main(String... args) throws Exception {
     // Instantiates a client
-    SpeechClient speech = SpeechClient.create();
+    try (SpeechClient speechClient = SpeechClient.create()) {
```
Comment (lesv) [inline speech/cloud-client/src/main/java/com/example/speech/QuickstartSample.java]: we use speech everywhere else, why not keep it consistent.  (don't really care which one you choose, but speech is shorter.

File: speech/cloud-client/src/main/java/com/example/speech/Recognize.java [hunk 3]
```diff
@@ -80,166 +86,172 @@ public static void main(String... args) throws Exception {
...
+      OperationFuture<LongRunningRecognizeResponse, LongRunningRecognizeMetadata, Operation> response =
+          speech.longRunningRecognizeAsync(config, audio);
+
+      while (!response.isDone()) {
+        System.out.println("Waiting for response...");
+        Thread.sleep(10000);
+      }
 
-    List<SpeechRecognitionResult> results = response.get().getResultsList();
+      List<SpeechRecognitionResult> results = response.get().getResultsList();
```
Comment (lesv) [inline speech/cloud-client/src/main/java/com/example/speech/Recognize.java]: I thought the new standard for these things was just to show the top one?

File: speech/cloud-client/src/test/java/com/example/speech/RecognizeIT.java [hunk 4]
```diff
@@ -57,52 +61,82 @@ public void tearDown() {
...
+    Recognize.streamingRecognizeFile(audioFileName);
     String got = bout.toString();
     assertThat(got).contains("how old is the Brooklyn Bridge");
   }
+
+  @Test
+  public void testAutomaticPunctuationFile() throws Exception {
+    Recognize.transcribeFileWithAutomaticPunctuation(audioFileName);
+    String got = bout.toString();
+    assertThat(got).contains("How");
```
Comment (lesv) [inline speech/cloud-client/src/test/java/com/example/speech/RecognizeIT.java]: I bet this turns out to be too brittle, but LGTM for now

**Pull request closed without merging**